PF/assignment2/prg1.java

```diff
@@ -0,0 +1,81 @@
+package assignment2;
+
+public class prg1 {
+	public static void main (String args[])
+	{
+		prg1 obj= new prg1();
+		
+		int i;
+		for(i=1;i<10;i++)
+		{
+			String a="";
+			a= obj.pyramid(i,9);
+			
+			System.out.println(a);
+		}
+	}
+	
+	public String spaces(int row,int n)
+	{
+	String a= "";
+	int i;
+				if(row<(n+1)/2)
+				{
+			for(i=5;i>row;i--)
```

Comment:
ACTION: Remove code duplicacy. You can do it by single while loop. Like - 

int i,j;
			if(row<(n+1)/2)
			{
				i = 5;
				j= row;
			}
			else
			{
				i= row-4;
				j=1;
			}
			
			while(i>j)
			{
				a= a + " "+ " ";
				i--;
			}